tests/connection_error.test.js

```diff
@@ -0,0 +1,23 @@
+const test = require('tape');
+const { init } = require('./server');
+
+(async () => {
+  const server = await init();
+
+  test('GET /logs as fast as you can!', async function (t) {
+    const response = await server.inject('/logs');
+    t.equal(response.statusCode, 200, '/logs visited');
+    t.end();
+  });
+  
+  test('GET /logs try it again!', async function (t) {
+    const response = await server.inject('/logs');
+    t.equal(response.statusCode, 200, '/logs visited');
+    t.end();
```

Comment:
it seems that the stop event code branch isn't invoked. this might help
```suggestion
    server.stop(function(){});
    t.end();
```

Reply:
Now, hapi has different stop function
https://hapi.dev/api/?v=19.1.1#server.stop()